src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -371,6 +372,16 @@ public async Task RemoveAccountAsync(ApplicationConfiguration appConfig, IAccoun
                         }
                         else
                         {
+                            _logger.Info("[WamBroker] Processing WAM exception");
+                            _logger.Verbose($"[WamBroker] TelemetryData: {result.TelemetryData}");
+
+                            string errorMessage =
+                                $" [WamBroker] \n" +
+                                $" Error Code: {result.Error.ErrorCode} \n" +
+                                $" Error Message: {result.Error.Context} \n" +
+                                $" Internal Error Code: {result.Error.Tag.ToString(CultureInfo.InvariantCulture)} \n";
+
+                            _logger.Error($"[WamBroker] Failed to sign out account. { errorMessage}");
```

Comment:
I think you may want to modify the new line locations. Wouldn't this print
`[WamBroker] Failed to sign out account. [WamBroker]` 
as the first line?

Maybe line 379 should be 
```CSharp
$" \n [WamBroker] Error Code: {result.Error.ErrorCode} \n" +
```

or maybe you can modify line 384 some how.

Reply:
updated, good catch.  will check other places as well.